parcels/particle.py

```diff
@@ -117,6 +117,7 @@ class ScipyParticle(_Particle):
     lon = Variable('lon', dtype=np.float32)
     lat = Variable('lat', dtype=np.float32)
     time = Variable('time', dtype=np.float64)
+    trajectory = Variable('trajectory', dtype=np.int32)
```

Comment:
I think I've fixed this now. The trick was to, when the list of variables is compiled in particle.py, first list all the 64-bit variables, and only then concatenate the 32-bit variables. In that way, we will never get a 64-bit variable to be misaligned